src/components/SliderContainer.ts

```diff
@@ -0,0 +1,198 @@
+import { Component, createElement } from "react";
+
+import { Slider } from "./Slider";
+
+interface SliderContainerProps {
+    mxObject: mendix.lib.MxObject;
+    valueAttribute: string;
+    maxAttribute: string;
+    minAttribute: string;
+    onChangeMicroflow: string;
+    stepValue: number;
+    stepAttribute: string;
+    noOfMarkers: number;
+    tooltipText: string;
+    decimalPlaces: number;
+    readOnly: boolean;
+}
+
+interface SliderContainerState {
+    maximumValue?: number;
+    minimumValue?: number;
+    value?: number;
+    stepValue?: number;
+}
+
+class SliderContainer extends Component<SliderContainerProps, SliderContainerState> {
+    private subscriptionHandles: number[];
+
+    constructor(props: SliderContainerProps) {
+        super(props);
+
+        this.state = {
```

Comment:
Very, very [similar ](https://en.wikipedia.org/wiki/Don't_repeat_yourself)to `updateValues`.